storops/unity/resource/user_quota.py

```diff
@@ -0,0 +1,128 @@
+
+    Operations supported:
+    create: Create user_quota on a file_system/tree_quota
+    modify: Modify user_quota on a file_system/tree_quota
+    """
+    @classmethod
+    def create(cls, cli, file_system_id=None, tree_quota_id=None,
+               hard_limit=None, soft_limit=None, uid=None,
+               unix_name=None, win_name=None):
+        """
+        Creates user_quota on the speficied filesystem or tree quota
+        :param file_system_id: This is an optional parameter which if provided
+            will create user_quota on the file_system specified. file_system_id
+            and tree_quota_id cannot be given together
+        :param tree_quota_id: This is an optional parameter which if provided
+            will create user_quota on the tree_quota specified. file_system_id
+            and tree_quota_id cannot be given together
+        :param hard_limit: sets hard_limit  on the file_system for the user
+            specified
+        :param soft_limit: sets soft_limit  on the file_system for the user
+            specified
+        :param uid: This is an optional parameter to specify user. Either of
+            uid/unix_name/win_name should be provided
+        :param unix_name: This is an optional parameter to specify user. Either
+            of uid/unix_name/win_name should be provided
+        :param win_name:This is an optional parameter to specify user. Either
+            of uid/unix_name/win_name should be provided
+        :return: created user quota.
+        """
+        fs_clz = storops.unity.resource.filesystem.UnityFileSystem
+        file_system = fs_clz.get(cli, file_system_id).verify()
+        user_quota_param = cls.prepare_user_quota_create_parameters(
+            file_system, tree_quota_id, hard_limit, soft_limit, uid,
+            unix_name, win_name)
+        resp = cli.post(
+            cls().resource_class, **user_quota_param)
+        resp.raise_if_err()
+        return cls(_id=resp.resource_id, cli=cli)
+
+    @classmethod
+    def modify(cls, cli, user_quota_id, hard_limit=None, soft_limit=None):
+        """
+        Modifies user_quota params for the specified user_quota_id
+        :param user_quota_id: This is required which speicfies user_quota to
+            be modified
+        :param hard_limit: modifies hard_limit  on the file_system for the
+            user_quota specified
+        :param soft_limit: modifies soft_limit  on the file_system for the
+            user_quota specified
+        :return: created user quota.
+        """
+        req_body = cli.make_body(
+            allow_empty=False,
+            hardLimit=hard_limit,
+            softLimit=soft_limit
+           )
+        resp = cli.action(cls().resource_class, user_quota_id,
+                          'modify', **req_body)
+        resp.raise_if_err()
+        return resp
+
+    @staticmethod
+    def prepare_user_quota_create_parameters(
+                  file_system, tree_quota, hard_limit,
```

Comment:
It is better to use filesystem, we also consider filesystem as one word in Unity code base.